### ISSUE TITLE
Errors in topips()

### ISSUE THREAD

**Issue by adamyala** [rshipp/python-dshield#5]
I use `pip install dshield` then ran:
```
>>> import dshield
>>> print(dshield.topips())
```

This is the error in 2.7:
```
>>> import dshield
>>> print(dshield.topips())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/my/path/here/lib/python2.7/site-packages/dshield.py", line 157, in topips
    return _get(uri, return_format)
  File "/my/path/here/lib/python2.7/site-packages/dshield.py", line 35, in _get
    return _strip_and_reformat(requests.get(''.join([__BASE_URL, function, JSON])).json())
  File "/my/path/here/lib/python2.7/site-packages/dshield.py", line 21, in _strip_and_reformat
    data_copy = data.copy()
AttributeError: 'list' object has no attribute 'copy'
```
I fixed this by adding be below snippt at line 21:
```python
    if type(data) == dict:
        data_copy = data.copy()
    elif type(data) == list:
        data_copy = list(data)
```
to maintain the shallow copy. I also added `TypeError` to the execptions on line 25 of `dshield.py`. 

This is the error in 3.5:
```
>>> import dshield
>>> print(dshield.topips())
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/my/path/here/lib/python3.5/site-packages/dshield.py", line 157, in topips
    return _get(uri, return_format)
  File "/my/path/here/lib/python3.5/site-packages/dshield.py", line 35, in _get
    return _strip_and_reformat(requests.get(''.join([__BASE_URL, function, JSON])).json())
  File "/my/path/here/lib/python3.5/site-packages/dshield.py", line 23, in _strip_and_reformat
    data_copy.__delitem__('METAKEYINFO')
TypeError: list indices must be integers or slices, not str
```
I remedied this by adding `AttributeError` to the execptions on line 25 of `dshield.py`.

Am I using this incorrectly or is this a current bug? If it's a bug I'm happy to make a PR.

**Comment by rshipp** [comment]
Thanks for reporting this. Looks like the API changed from returning ordered dicts to lists. You can see the old expected format in the TestInternals cases in test.py. The "correct" fix will be a little more involved than your patches, as the exceptions you mentioned are just symptoms of the upstream change. You're welcome to do a PR, or I'll probably fix this later today.

**Comment by adamyala** [comment]
If you've got spare time to make the fix today, I'll leave it to you as you know the code base better than I do.

Thank you for building and maintaining this.

**Comment by rshipp** [comment]
Should be fixed now, let me know if you have any more problems.

**Comment by rshipp** [comment]
Fixed in 76996a72f3c27c5208496ff3a6c472d2cb13e5e7
